daq_macros.py

```diff
@@ -1909,6 +1915,7 @@ def snakeRasterBluesky(rasterReqID, grain=""):
     initiate transitions here allows for GUI sample/heat map image to update
     after moving to known position"""
     logger.debug(f'lastOnSample(): {lastOnSample()} autoRasterFlag: {autoRasterFlag}')
+    yield from bps.sleep(3) #waiting for detector to not lose row
```

Comment:
this would be a major problem for the other beamlines, where this sleep isn't necessary